app/src/main/resources/com/oracle/javafx/scenebuilder/app/DocumentWindow.fxml

```diff
@@ -74,6 +74,7 @@
                               <RadioMenuItem fx:id="libraryViewAsSections" mnemonicParsing="false" onAction="#onLibraryViewAsSections" text="%library.panel.menu.view.sections" toggleGroup="$libraryDisplayOptionTG" />
                             <SeparatorMenuItem mnemonicParsing="false" />
                             <MenuItem mnemonicParsing="false" onAction="#onManageJarFxml" text="%library.panel.menu.manage.jar.fxml" />
+                            <MenuItem mnemonicParsing="false" onAction="#onLibraryRefresh" text="%library.panel.menu.refresh" />
```

Comment:
Please update the license header in line 4 from `Copyright (c) 2016, 2019, Gluon and/or its affiliates.` to `Copyright (c) 2016, 2022, Gluon and/or its affiliates.`.